### PR TITLE
Adding saba version 0.1

### PR DESCRIPTION
**Pull request by bsipocz**
Since saba already lives under the astropy gh account, adding it to the conda channel makes sense I think.

cc @nocturnalastro

### CODE REVIEW

**Comment by nocturnalastro** [comment]
@bsipocz, I agree is there anything you need me to do?

**Comment by bsipocz** [comment]
Hopefully not, travis should pass and then we're all set.

**Comment by mwcraig** [comment]
Looks like this will need a recipe template because the runtime `numpy` dependency in saba isn't declared in setup. @nocturnalastro -- if you are amenable you could add `numpy` for the next  saba release.

**Comment by nocturnalastro** [comment]
@mwcraig  does it need to be put in the setup.py? `numpy` in pip-requirements.

I need to update the requirements soon anyway as the pip-requirements needs to be directed astropy master. Not my branch now that the plugin code is merged. So I'll replace the pip package.

**Comment by bsipocz** [comment]
@mwcraig - I tried to give a go to ``conda skeleton pypi saba``, but get all kind of error. 
Do you use that to create the skeletons, or is there some other tricks?

**Comment by bsipocz** [comment]
@nocturnalastro - Do you mind if I open a small cleanup PR to saba? it seems that there unnecessary many requirements/install files there, and things could be simplified a bit.

**Comment by nocturnalastro** [comment]
@bsipocz - Go ahead :) any improvements are always welcome.

**Comment by bsipocz** [comment]
I guess I need to add the sherpa channel to the dependencies. @mwcraig is it possible to do that only for saba, or should it be the global ``--inspect-channels``?

**Comment by mwcraig** [comment]
It isn't possible on a per-package basis, so try adding it to `--inspect-channels` and we'll see what happens.

Please add it *before* conda-forge is added so that we end up with this priority order:

+ defaults
+ conda-forge 
+ sherpa

File: recipe_templates/saba/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install --single-version-externally-managed --record record.txt
```
Comment (mwcraig) [inline recipe_templates/saba/bld.bat]: also need `--offline`

File: recipe_templates/saba/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record record.txt
```
Comment (mwcraig) [inline recipe_templates/saba/build.sh]: Add `--offline`

File: requirements.yml [hunk 3]
```diff
@@ -199,6 +199,12 @@
   excluded_platforms:
       - 'win-64'
 
+- name: saba
+  version: '0.1'
+  setup_options: '--offline'
```
Comment (mwcraig) [inline requirements.yml]: FYI, this is used only when the recipe is generated from skeleton.

**Comment by bsipocz** [comment]
Back to square one. However in the meantime I think I've fixed most of the issues in saba, so maybe it's easier to fix the conda build with a new release :)

**Comment by mwcraig** [comment]
Interesting...apparently numpy is a build dependency too, I didn't expect that. Feel free to try adding it to build, otherwise it is fine to wait until a new release is out.

**Pull request closed without merging**